Workshop.md

```diff
@@ -0,0 +1,479 @@
+# Building a "ToDo" Backend with Kitura
+
+<p align="center">
+<img src="https://www.ibm.com/cloud-computing/bluemix/sites/default/files/assets/page/catalog-swift.svg" width="120" alt="Kitura Bird">
+</p>
+
+<p align="center">
+<a href= "http://swift-at-ibm-slack.mybluemix.net/"> 
+    <img src="http://swift-at-ibm-slack.mybluemix.net/badge.svg"  alt="Slack"> 
+</a>
+</p>
+
+## Run the tests
+In order to implement a ToDo Backend, a server is required that provides support for storing, retrieving, deleting and updating "to do" items. The ToDoBackend project doesn't provide a specification for how the server must respond, rather it provides a set of tests which the server must pass. The "todo-backend-js-spec" project provides those tests.
+
+### 1. Run the ToDo-Backend Tests:
+
+1. Open the tests in a web browser: `open ~/todo-backend-js-spec/index.html`
+2. Set a "test target root" of `http://localhost:8080`
+3. Click "run tests".
+
+All the tests should fail. The first error reported should be as follows:
+
+:x: `the api root responds to a GET (i.e. the server is up and accessible, CORS headers are set up)`
+
+```
+AssertionError: expected promise to be fulfilled but it was rejected with [Error: 
+
+GET http://localhost:8080/
+FAILED
+
+The browser failed entirely when make an AJAX request.
+```
+
+This shows that the tests made a `GET` request to `http://localhost.com:8080`, but it failed with no response. This is expected as there is no server yet.
+
+In the instructions below, reloading the page will allow you to re-run the ToDo-Backend tests.
+
+## Building a Kitura server
+
+Implementing a compliant ToDo Backend is an incremental task, with the aim being to pass more of the testsuite at each step. The first step is to build and run your Kitura server to respond to requests.
+
+### 1. Run your Kitura server
+
+1. Open the ToDoServer project in Xcode. 
+    ```
+    cd ~/ToDoBackend/ToDoServer  
+    open ToDoServer.xcodeproj
```

Comment:
There isn't an .xcodeproj file generated if you follow Option 1: Create your Kitura starter kit in IBM Cloud in the README